src/js/client/src/Pages/characters/CharacterView.tsx

```diff
@@ -0,0 +1,67 @@
+import { createResource, createSignal, JSXElement, Show, Suspense } from "solid-js";
+import { Character, Details, Stats } from "../../../../Shared/Models/index";
+
+
+
+type CharacterViewProps = {
+    id: number
+}
+
+const fetchCharacter = async (id : number) => {
+
+    const res = (await fetch(`http://localhost:8080/character/${id}/`)).json();
```

Comment:
Vi skal nok på et tidspunkt have lagt fetch i en separat fil (tænker ikke at alle skal være i samme fil, men de kan jo grupperes efter hvad der lige giver mening). Jeg tænker ikke at det er super relevant lige nu, da vi nok under alle omstændigheder kommer til at rafinere lidt mere hvordan kommunikation mellem frontend og backend fungerer.

Reply:
Præcis. Tænkte bare at sætte et proof of concept op :D